openfe/storage/resultserver.py

```diff
@@ -0,0 +1,65 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/gufe
```

Comment:
wrong link here, but I suspect there's a few files either missing or with the wrong license header since we're switching things quite liberally